docs/root/configuration/operations/overload_manager/overload_manager.rst

```diff
@@ -102,6 +109,39 @@ The following overload actions are supported:
     - Envoy will reset expensive streams to terminate them. See
       :ref:`below <config_overload_manager_reset_streams>` for details on configuration.
 
+
+Load Shed Points
+----------------
+
+Load Shed Points are similar to overload actions as they are dependent on a
+given trigger to activate which determines whether Envoy ends up shedding load at
+the given point in a connection or stream lifecycle.
+
+For a given request on a newly created connection, we can think of the
+configured load shed points as a decision tree at key junctions of a connection
+/ stream lifecycle. While a connection / stream might pass one junction, it
+is possible that later on the conditions might change causing Envoy to shed load
+at a later junction.
+
+In comparision to analogous overload actions, Load Shed Points are more
+reactive to changing conditions, especially in cases of large traffic spikes.
+
+Compared to overload actions, Load Shed Points are also more flexible to
+integrate custom (e.g. company inteneral) Load Shed Points as long as the extension
+has access to the Overload Manager to request the custom Load Shed Point.
```

Comment:
nit: Are there cases that Overload-Actions are preferred over Load-Shed-Points?
If so, maybe add a short sentence/example where this holds.

Reply:
Good idea, added more on when each solution is better suited to the a situation.